experiments/generic/node_memory_hog/node_memory_hog_k8s_job.yml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: node-memory-hog-
+spec:
+  template:
+    metadata:
+      labels:
+        experiment: node-memory-hog
+    spec:
+      serviceAccountName: %CHAOS_SERVICE_ACCOUNT%
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: litmuschaos/ansible-runner:ci
+        imagePullPolicy: Always
+        env: 
+          - name: ANSIBLE_STDOUT_CALLBACK
+            value: 'default'
+
+          ## ENTER THE NAMESPACE WHERE THE APPLICATION IS RUNNING
+          - name: APP_NAMESPACE
+            value: ''
+
+          ## ENTER THE LABEL OF THE APPLICATION
+          - name: APP_LABEL
+            value: ''
+          
+          ## ENTER THE KIND OF APPLICATION 
+          ## WORKING FOR 'deployment' TYPE OF APPLICATION
+          - name: APP_KIND
+            value: ''
+          
+          ## PLATFORM SUPPORTED 
+          - name: PLATFORM
+            value: 'GKE'
+          
+          ## ENTER TOTAL_CHAOS_DURATION(IN sec)
+          ## THIS MAY CHANGES (LESS THAN GIVEN VALUE) DUE TO NO RESOURCE LEFT IN THE SYSTEM
+          ## DEFAULT VALUE IS 60
+          - name: TOTAL_CHAOS_DURATION
+            value: '60'
+
+          ## ENTER THE MEMORY YOU WANT TO HOG ON NODE (In Gigabytes G)
```

Comment:
In some places, you have used all caps for comment and at some places, you have put all small. Can you please make them uniform?

Reply:
done